packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -266,6 +269,8 @@ export enum TelemetryProperty {
   DocumentationName = "documentation-name",
   // Used with OpenNewProject
   VscWindow = "vscode-window",
+  // Used with SelectFolder
+  Option = "option",
```

Comment:
Can we make it more specific to avoid confusion?

Reply:
No problem.